snapshot.yaml

```diff
@@ -10,16 +10,21 @@ packages:
   - HPDF-1.5.1
   - odd-jobs-0.2.2
   - patrol-0.0.1
-  - persistent-2.11.0.2
-  - persistent-template-2.9.1.0
-  - persistent-mongoDB-2.11.0.0
-  - persistent-postgresql-2.11.0.1
   - prolude-0.0.0.17
   - servant-lucid-0.9.0.2
   - servant-serf-0.1.0
   - servant-static-th-1.0.0.0
   - timing-convenience-0.1
 
+  # https://github.com/yesodweb/persistent/pull/1209
+  - git: https://github.com/yesodweb/persistent
```

Comment:
We're going to need to update the snapshot otherwise this will be building persistent on ci every time.

Reply:
This is the snapshot?

Reply:
Yeah this is the snapshot. I don't blame him though, snapshot YAML and `stack.yaml` are identical!

Reply:
Hah! I read smurf instead of snapshot

Reply:
EdutainmentLIVE/smurfshot